omegaconf/basecontainer.py

```diff
@@ -246,7 +250,7 @@ def get_node_value(key: Union[DictKeyType, int]) -> Any:
 
         if isinstance(conf, DictConfig):
             if (
-                conf._metadata.object_type is not None
+                conf._metadata.object_type not in (dict, None)
```

Comment:
for my reference, this is correct as tuple.__contains__() does "a is b or a == b"